tests/app/validation/test_number_validator.py

```diff
@@ -3,15 +3,16 @@
 
 from app.validation.error_messages import error_messages
 from app.validation.validators import NumberCheck, DecimalPlaces
-from wtforms.validators import StopValidation, ValidationError
 from app.forms.fields import get_number_field, CustomDecimalField, MAX_DECIMAL_PLACES
 from app.data_model.answer_store import AnswerStore
+from wtforms.validators import StopValidation, ValidationError
 
 
-class TestNumberValidator(unittest.TestCase):
+class TestNumberValidator(unittest.TestCase):  # pylint: skip-file
```

Comment:
`skip-file` needs to be removed and replaced with the specific lint rule we're disabling